### ISSUE TITLE
Broken on MacOS

### ISSUE THREAD

**Issue by jmitchell89** [giann/settlers-issues#2]
Seg faults when trying to launch.

**Comment by giann** [comment]
What version of macOS do you have ? Do you have Löve or Lua already installed ? If so which versions ?

**Comment by jmitchell89** [comment]
I do not have Löve or Lua installed. I have Mojave 10.14.1

**Comment by giann** [comment]
Ok I reproduced the issue. Until I find how to fix it you can run the game like so:

Open a terminal and enter this command (assuming the game is in Applications)
```bash
/Applications/Settlers\ of\ the\ Deep.app/Contents/MacOS/love /Applications/Settlers\ of\ the\ Deep.app/Contents/Resources/Settlers\ of\ the\ Deep.love
```

**Comment by Davenport-Physics** [comment]
https://bitbucket.org/rude/love/issues/1442/v110-doesnt-work-on-macos-mojave
https://love2d.org/wiki/11.2

Make sure you're distributing the game with love version 11.2

**Comment by giann** [comment]
Yes I read that too. But I experienced some loss of fps with 11.2. I'll have to test it some more.

**Comment by Davenport-Physics** [comment]
Possibly related to this bug. It affects OS X 10.14.x

https://bitbucket.org/rude/love/issues/1455/l-ve-112-uses-55-of-cpu-power-on-no-game

There probably isn't much you can do unfortunately.

**Comment by giann** [comment]
macOS increasingly sucks this days... But if it was macOS fault I should have the same issue with Löve 11.1.

Meanwhile the solution I describe [here](https://github.com/giann/settlers-issues/issues/2#issuecomment-443986629) should work.

**Comment by giann** [comment]
Also I doubt the issue you linked is the one at play since vsync is disabled in this game.
